terraform/deployments/govuk-publishing-platform/security_groups.tf

```diff
@@ -4,10 +4,24 @@ resource "aws_security_group" "mesh_ecs_service" {
   name        = "mesh_ecs_service-${terraform.workspace}"
   vpc_id      = local.vpc_id
   description = "Associated with all ECS Services that are virtual services in the AppMesh mesh"
+
+  tags = merge(
+    local.additional_tags,
+    {
+      name = "mesh-ecs-service-sg"
+    },
+  )
 }
 
 resource "aws_security_group" "smokey" {
   name        = "ecs_fargate_smokey-${terraform.workspace}"
   vpc_id      = local.vpc_id
   description = "Smoke test runner"
+
+  tags = merge(
+    local.additional_tags,
+    {
+      name = "ecs-fargate-smokey-sg"
```

Comment:
it doesn't have the workspace interpolated in comparison to the previous tags above.